recipes-bsp/hm-legacy-names/hm-legacy-names.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "Legacy Network Names Script and Service"
```

Comment:
there is no service here only the script. the service is in hm-interface-names.bb. maybe consider hold it to one bb file in this case?

Reply:
The intention is that this recpipe  is only dependent on bash and
```RPROVIDES:${PN} = "hm-stable-names"``` which the other  ```RDEPENDS:${PN} = "hm-stable-names"```